package.json

```diff
@@ -14,9 +15,12 @@
     "#dev-util": "./dev/src/util.js"
   },
   "scripts": {
+    "commit": "cz",
     "cp": "mkdir -p ./dev/public/shiki && cp -r ./node_modules/shiki/{dist,languages,samples,themes} ./dev/public/shiki",
     "cp:shikifix": "cp node_modules/.pnpm/vscode-oniguruma@1.7.0/node_modules/vscode-oniguruma/release/onig.wasm ./dev/public/shiki/dist/",
     "dev": "vite dev dev --config ./vite.config.js",
+    "eik:login": "npx @eik/cli login",
```

Comment:
I'd prefer `@eik/cli` as a devDep instead and running this directly via `eik login` instead of `npx @eik/cli login` (same with the publish step).

Reply:
Fixed 👍